### PR TITLE
Implement user management

### PR DESCRIPTION
**Pull request by mgs95**

### CODE REVIEW

File: admin-api/adapter/service/proto/monitoringpb/monitoring.pb.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 	math "math"
 )
 
-// Reference imports to suppress errors if they are not otherwise used.
+// Reference imports to suppress httperrors if they are not otherwise used.
```
Comment (mgs95) [inline admin-api/adapter/service/proto/monitoringpb/monitoring.pb.go:16]: is this desired? (applies to line 626 too)

Comment (angelsanchez) [reply admin-api/adapter/service/proto/monitoringpb/monitoring.pb.go:16]: fixed

File: admin-ui/src/components/Layout/ModalContainer/layouts/ModalLayoutConfirmList/ModalLayoutConfirmList.module.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  margin: 10px 0px 25px 0;
+}
+
+.list {
+  max-height: 9 * 5 * $grid-unit;
+  overflow: auto;
+
+  & > div:not(:last-child) {
+    border-bottom: solid 1px palette(base, 300);
+    border-bottom: solid 1px palette(base, 300);
```
Comment (angelsanchez) [inline admin-ui/src/components/Layout/ModalContainer/layouts/ModalLayoutConfirmList/ModalLayoutConfirmList.module.scss:18]: duplicated line

File: admin-ui/src/pages/AddUser/AddUser.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  }, [register, setValue]);
+
+  useEffect(() => {
+    if (mutationError) {
+      setError('email', '', mutationError.toString());
+    }
+  }, [mutationError, setError]);
+
+  function onCompleteAddUser(updatedData: CreateUser) {
+    // TODO: CHECK FOR API ERRORS
```
Comment (angelsanchez) [inline admin-ui/src/pages/AddUser/AddUser.tsx:85]: remove this

File: admin-ui/src/pages/Settings/pages/UsersSettings/components/UserFiltersAndActions/UserActions.tsx [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+  </div>
+);
+const CustomSeparator: FC<CustomOptionProps> = ({ label }) => (
+  <div className={cx(styles.customOption, styles.separator)}>{label}</div>
+);
+
+enum Actions {
+  DELETE = 'DELETE',
+  REVOKE_ACCESS = 'REVOKE ACTIVE SESSIONS',
+  CHANGE_USER_TYPE_TO = 'CHANGE USER TYPE TO',
```
Comment (angelsanchez) [inline admin-ui/src/pages/Settings/pages/UsersSettings/components/UserFiltersAndActions/UserActions.tsx:56]:
```suggestion
  CHANGE_USER_TYPE_TO = 'CHANGE ACCESS LEVEL',
```

File: admin-ui/src/pages/Settings/pages/UsersSettings/components/UserList/UserList.tsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+      text: 'delete',
+      callToAction: (_: any, userId: string) => onDeleteUsers([userId])
+    },
+    {
+      Icon: IconRevoke,
+      text: 'revoke active sessions',
+      callToAction: (_: any, userId: string) => onRevokeUsers([userId])
+    },
+    {
+      text: 'change user type to',
```
Comment (angelsanchez) [inline admin-ui/src/pages/Settings/pages/UsersSettings/components/UserList/UserList.tsx:34]:
```suggestion
      text: 'change access level to',
```

**Pull request merged**